### PR TITLE
MUST NOT use GitHub secrets

### PR DESCRIPTION
**Pull request by pellared**
I think it was the original intention.

@owais @flands PTAL

### CODE REVIEW

File: specification/repository.md [hunk 1]
```diff
@@ -52,10 +52,8 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
 
 ### GitHub Actions
 
-- MUST use [GitHub
-  secrets](https://docs.github.com/en/actions/reference/encrypted-secrets) to
-  store sensitive data (auth tokens, passwords) and limit their usage to only
-  required places
+- MUST NOT use [GitHub
+  secrets](https://docs.github.com/en/actions/reference/encrypted-secrets)
```
Comment (mateuszrzeszutek) [inline specification/repository.md:56]: Does `GITHUB_TOKEN` count as a secret?
If you run tests using GHCR you *have* to use that

Comment (pellared) [reply specification/repository.md:56]: Good comment.
For sure not (it is not explicitly set by us). I will add a remark that it does not count

Comment (pellared) [reply specification/repository.md:56]: PTAL https://github.com/signalfx/gdi-specification/pull/186/commits/97830548256438eec54a815cfba27a3f57bfaf86

**Review by mateuszrzeszutek — APPROVED** [review]

**Comment by breedx-splk** [comment]
Noooo! Well, maybe this ship has already sailed, but I read this a different way. I read this to mean that "if you have a need for secrets in your repo, use GitHub secrets for these (instead of files or hard-coding or some other means)". I thought it was supporting/encouraging the use of GH secrets for sensitive data....but this change completely eliminates that....in fact now it's forbidden. 

I would like to see this change rolled back.

**Pull request merged**